### PR TITLE
Test ci

### PR DESCRIPTION
**Pull request by daniel-dinu**
CI using AppVeyor - #107

### CODE REVIEW

**Comment by khovratovich** [comment]
Could you give more details on this update?

**Comment by daniel-dinu** [comment]
Some details:
* I added a Visual Studio solution + several Visual Studio projects to build Argon2 on Windows.
* I added Windows PowerShell scripts to test against the KATs and to verify the checksums of these files.
* I configured AppVeyor, thus Continuous Integration for Windows.

Mainly it adds some missing functionalities for Windows, which were already available for Linux/OS X.

**Comment by technion** [comment]
The interface looks good, and we finally have a PR without all those crosses. Can I suggest squashing all those commits?

File: vs2015/Argon2OptTestCI/Argon2OptTestCI.vcxproj [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    <ClInclude Include="..\..\src\blake2\blamka-round-ref.h" />
+    <ClInclude Include="..\..\src\core.h" />
+    <ClInclude Include="..\..\src\encoding.h" />
+    <ClInclude Include="..\..\src\opt.h" />
+    <ClInclude Include="..\..\src\thread.h" />
+  </ItemGroup>
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
+  <ImportGroup Label="ExtensionTargets">
+  </ImportGroup>
+</Project>
```
Comment (Herry0092) [inline vs2015/Argon2OptTestCI/Argon2OptTestCI.vcxproj:159]:
```suggestion
</Project>
```

File: kats/check-sums.ps1 [hunk 2]
```diff
@@ -0,0 +1,42 @@
+Set-Variable tempfile -option Constant -value "tempfile"
+
+function hash($path) {
+    $fullPath = Resolve-Path $path
+    $hash = new-object -TypeName System.Security.Cryptography.SHA256CryptoServiceProvider
+
+    $contents = [IO.File]::ReadAllText($fullPath) -replace "`r`n?", "`n"
```
Comment (Herry0092) [inline kats/check-sums.ps1:7]: ``

**Review by Herry0092 — COMMENTED** [review]
[#385](url)

**Pull request merged**